### PR TITLE
Query  update animals table

### PR DESCRIPTION
**Pull request by Owusu-Desmond**
In this pull request,

- [ ] I have to insert some data into the table and then I have to update the table by setting the species column to unspecified. I have to verify that the change was made. Then I have to roll back the change and verify that the species columns went back to the state before the transaction.

this is the screenshot of the result of the query

Screenshot from 2022-09-13 14-14-16

- [ ] I have to update the `animals` table by setting the species column to `digimon` for all animals that have a name ending in mon. I have to update the `animals` table by setting the species column to pokemon for all animals that don't have species already set. I have to commit the transaction. I have to verify that the change was made and persists after commit.

this is the screenshot of the result of the query
Screenshot from 2022-09-13 14-19-45

- [ ] I have to delete all records in the `animals` table, then roll back the transaction. After the rollback verify if all records in the `animals` table still exists.

this is the screenshot of the result of the query
Screenshot from 2022-09-13 14-23-26
of the result of the query

- [ ] I have to delete all animals born after Jan 1st, 2022. I have to create a savepoint for the transaction. I have to update all animals' weights to be their weight multiplied by -1. I have to `rollback` to the savepoint. I have to update all animals' weights that are negative to be their weight multiplied by -1. I have to commit the transaction.

this is the screenshot of the result of the query
Screenshot from 2022-09-13 14-30-53

- [ ] This is the screenshot questions and answers of the queries I have to write

Screenshot from 2022-09-13 14-46-52

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species TEXT,
```
Comment (gemmen29) [inline schema.sql]: - [ ] Kindly add the species column using the `alter` statement as the table is already created.

File: queries.sql [hunk 2]
```diff
@@ -25,4 +25,93 @@ SELECT * FROM animals WHERE neutered = true;
...
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT savepoint_name;
+
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO savepoint_name;
+
+UPDATE animals SET weight_kg = weight_kg * -1;
```
Comment (gemmen29) [inline queries.sql]: - [ ] Kindly update the query after rollback in that transaction to meet the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Update%20all%20animals%27%20weights%20that%20are%20negative%20to%20be%20their%20weight%20multiplied%20by%20%2D1.). They asked only to update weights that have a negative value, so you need to use `where` clause.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Owusu-Desmond, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @Owusu-Desmond 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**